src/Symfony/Component/Yaml/Inline.php

```diff
@@ -591,6 +591,7 @@ private static function evaluateScalar($scalar, $flags, $references = array())
                     case 0 === strpos($scalar, '!!binary '):
                         return self::evaluateBinaryScalar(substr($scalar, 9));
                     case preg_match('/^(-|\+)?[0-9][0-9,]*(\.[0-9_]+)?$/', $scalar):
+                        @trigger_error('Using the comma as a group separator for floats is deprecated since version 3.2 and will be removed in 4.0.', E_USER_DEPRECATED);
```

Comment:
@Taluu You are right. I opened #18858 which should fix this issue.

> And I think a `_` may have been forgotten in the part before the `.` actually.

I think it's okay the way it is. We only introduced the `_` for 3.2 and I think people should not start mixing `_` and `,` then. If they want to use `_`, they should just move away from `,` as well.